spec/support/schemas/payment_information.json

```diff
@@ -15,10 +15,10 @@
                         "not_deceased_indicator": true
                     },
                     "payment_account": {
-                        "account_number": "9876543211234",
```

Comment:
Must have missed this before myself 😄but this file should be a json-schema definition rather than an expectation for the values.

Reply:
Oh man, you are correct. Will fix haha

Reply:
Alright, now it checks the payload AND validates the schema.